CHANGELOG.md

```diff
@@ -2,6 +2,94 @@
 
 ## Release 2025-01-06
 
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as `v20250130`.
+
+### Announcements
+* General support for AKS Kubernetes version 1.28 was deprecated on Jan 30, 2025. Upgrade your clusters to version 1.29 or later. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
+* Security patch releases in release tracker, starting with 20250115T000000Z will contain release notes for the release.
+
+### Release Notes
+* Features:
+  * AKS Kubernetes patch versions 1.29.11, 1.30.7 and 1.31.3 are now available.
+  * AKS LTS version 1.28 available in all regions.
+  * The upcoming apiVersion 2025-01-01 will enable additional outbound configurations to allow for servicing of network isolated clusters.
```

Comment:
this seems like an announcement

Reply:
IDK - when we light up the apiVersion, this will be a feature but we won't have this level of detail (i.e. this specific change) when that apiVersion goes live.  I think feature is right here.

Reply:
reworded for future proofing